test/server/cards/03_TWI/units/JesseHardFightingPatriot.spec.ts

```diff
@@ -0,0 +1,26 @@
+describe('Jesse, Hard-Fighting Patriot', function () {
+    integration(function (contextRef) {
+        describe('Jesse\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['jesse#hardfighting-patriot']
+                    }
+                });
+            });
+
+            it('should create two Battle Droid tokens for the opponent', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.jesse);
+                expect(context.player2).toBeActivePlayer();
+
+                const battleDroids = context.player2.findCardsByName('battle-droid');
+                expect(battleDroids.length).toBe(2);
+                expect(battleDroids).toAllBeInZone('groundArena');
```

Comment:
Just to be extra explicit, let's specify the player parameter here: `expect(battleDroids).toAllBeInZone('groundArena', context.player2);`

By default the `toBeInZone` expectations will check that the unit is in the arena of its controller, which is a good default. Since there's some potential issues around controllership here it's good to be explicit though I think.

Reply:
added